doc/specs/#5000 - Process Model 2.0/#5000 - Process Model 2.0.md

```diff
@@ -334,12 +334,12 @@ commandline in a given WT window?
 
 In addition to the concept of Window and Content Processes, we'll also be
 introducing another type of categorization for window processes. These are
-"Monarch" and "Servant" processes. This will allow for the coordination across
+"Monarch" and "Peasant" processes. This will allow for the coordination across

```

Comment:
Consider using "Subject" instead of "Peasant." A monarch has subjects, of which _some_ may be considered peasants. It's also less silly to say :D

Reply:
Is there a technical reason not to call it by an established term like primary/secondary or leader/follower?

Reply:
Because they're stuck on the name Monarch.

Reply:
Sorry for asking, but why are they stuck and what's the precedent? 🙈🙈🙈
I've been trying to find sources for "monarch" terminology in the context of leader-processes and I wasn't able to find anything, neither in this project nor in the wider web.

Reply:
I have seen it used in window hierarchy models before, but I can't remember where.